OracleJava/README.md

```diff
@@ -1,21 +1,21 @@
 Oracle Java on Docker
 =====
-This repository contains sample Docker configurations to facilitate installation and environment setup for DevOps users. This project includes Dockerfiles based on Oracle Linux with JDK images of currently supported versions 11 and later and for Server JRE 8.
+This repository contains sample Docker configurations to facilitate installation and environment setup for DevOps users. This project includes Dockerfiles based on Oracle Linux with JDK images of JDK 16, 15, and 11 and for Server JRE 8.
```

Comment:
Please change "Docker" to "container". You can keep Dockerfiles though, but wrap it in backtick quotes.

Reply:
Done

Reply:
Replaced Docker with container. Added backtick quotes to Dockefile  when using it as part of a command.